src/libraries/System.IO.Compression.ZipFile/src/System.IO.Compression.ZipFile.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser;$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Browser;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
```

Comment:
This makes sure that the Unix is picked as the default.

```suggestion
    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Browser;$(NetCoreAppCurrent)</TargetFrameworks>
```

Reply:
Shouldn't we also remove '$(TargetPlatformIdentifier)' == 'Unix'` if we make this change

Reply:
Ok, thanks pushing the changes.